tests/mock_callable_testslide.py

```diff
@@ -860,7 +879,9 @@ def can_not_mock(self):
                 mock_callable(self.target_arg, self.callable_arg)
 
     @context.shared_context
-    def async_methods_examples(context, not_in_class_instance_method=False):
+    def async_methods_examples(
+        context, not_in_class_instance_method=False, alllow_private=True
```

Comment:
Do we need this?

Reply:
Not. stayed there by accident, removing it rn.